sources/models/shared-defaults/kubernetes-services.toml

```diff
@@ -36,6 +37,10 @@ template-path = "/usr/share/templates/kubernetes-ca-crt"
 path = "/etc/systemd/system/kubelet.service.d/exec-start.conf"
 template-path = "/usr/share/templates/kubelet-exec-start-conf"
 
+[configuration-files.credential-provider-config-yaml]
+path = "/etc/kubernetes/kubelet/credential-provider-config.yaml"
+template-path = "/usr/share/templates/credential-provider-config-yaml"
+
```

Comment:
This needs an `AddPrefixesMigration` to clean up the configuration-files entry - search in the archived migrations for related changes.